pkg/netpol/connlist/exposed_peer.go

```diff
@@ -34,5 +32,5 @@ type XgressExposureData interface {
 	// PodLabels are matchLabels of potential pods which the peer might be exposed to
 	PodLabels() map[string]string
 	// PotentialConnectivity the potential connectivity of the exposure
-	PotentialConnectivity() conn.AllowedSet
+	PotentialConnectivity() common.Connection
```

Comment:
I'm not sure want the return value to be an interface type from internal package. 
Please add another sub-task to address this later.

Reply:
a solution I could think of is to expose common.Connection as connlist.AllowedSet ( so we don't duplicate the interface implementation again)  - done on [commit](https://github.com/np-guard/netpol-analyzer/pull/298/commits/bd98185794a50023ff2d74db9c1307007dd681aa)

Reply:
let's revert this commit and think about other options later.. should be done in a separate sub task